### DISCUSSION TITLE
Italics for Pretendard Std?

### DISCUSSION THREAD

**Discussion by cssobral2013** [orioncactus/pretendard#69] [Q&A] [answered]
And thank you for making the Std (Latin/Greek/Cyrillic-only) version!

**Comment by kms0219kms** [comment]
Hello, thank you for using our fonts and for giving us your feedback.
Do you have a problem we can help with?

**Comment by kms0219kms** [comment]
If you have an issue that you need help with, please leave a reply here and we'll do our best to help.

Please close this issue if there is no problem to help, or if it has already been resolved.

**Comment by orioncactus** [comment, marked as answer]
Hello! Sadly I don't have a plan for an Italic style of Pretendard at the moment. However I plan to monitor Pretendard's usage trends and proceed if necessary.
And I'll move this issue to discussion. So if you have any further questions, please reply the comment on Discussion. Thank you!

**Reply by kms0219kms** [reply]
Note that although it does not provide its own italic glyphs,
For the software's own italic, it works fine.

It would be good to take advantage of it and use it.

**Comment by cssobral2013** [comment]
@orioncactus & @kms0219kms - I decided to fork this font. Thank you, despite that not being what I wanted to hear.
